cocos/base/CCDirector.cpp

```diff
@@ -534,7 +562,7 @@ void Director::multiplyMatrix(MATRIX_STACK_TYPE type, const Mat4& mat)
     }
     else if(MATRIX_STACK_TYPE::MATRIX_STACK_PROJECTION == type)
     {
-        _projectionMatrixStack.top() *= mat;
+        _projectionMatrixStackList[0].top() *= mat;
```

Comment:
the same comment should be added in other functions, such as `Director::loadMatrix(MATRIX_STACK_TYPE type, const Mat4& mat)`.